The crypto certs we ship in Solaris are in pkg:/crypto/ca-certificates
and installed under /etc/certs/CA & /etc/openssl/certs/. The Solaris
equivalent to the filenames used in this chunk of the configure file
is /etc/certs/ca-certificates.crt which is generated by
/lib/svc/method/svc-ca-certificates from the others.

This patch should be sent upstream.

--- pidgin-2.10.11/configure.ac.orig	2015-11-24 06:27:37.749762307 -0800
+++ pidgin-2.10.11/configure.ac	2015-11-24 06:28:34.540905669 -0800
@@ -1057,7 +1057,8 @@
 			/etc/ssl/certs/ca-certificates.crt \
 			/etc/pki/tls/cert.pem \
 			/usr/local/share/certs/ca-root-nss.crt \
-			/etc/ssl/cert.pem
+			/etc/ssl/cert.pem \
+			/etc/certs/ca-certificates.crt
 		do
 			if test -e $i; then
 				gg_gnutls_sts="$i"
